testdata/p4_16_samples_outputs/pna-example-varIndex-1.p4.spec

```diff
@@ -18,14 +18,14 @@ header vlan_tag_1 instanceof vlan_tag_h
 struct main_metadata_t {
 	bit<16> pna_pre_input_metadata_parser_error
 	bit<32> pna_main_input_metadata_input_port
-	bit<32> local_metadata_depth
+	bit<8> local_metadata_depth
```

Comment:
Where are all these changes coming from? Are they correct?

Reply:
Previous implementation is as below:

int getMetadataFieldWidth(int width) {
    if (width % 8 != 0) {
        BUG_CHECK(width <= 64, "Metadata bit-width expected to be within 64-bits, found %1%",
                  width);
        if (width < 32)
            return 32;
        else
            return 64;
    }

Now i changed to nearest multiple of 8bits instead of 32 or 64bits. This covers bits widths of more than 64bits as well. I don't think anything wrong here.